src/debugpy/launcher/handlers.py

```diff
@@ -10,6 +10,8 @@
 from debugpy.common import json
 from debugpy.launcher import debuggee
 
+# see https://github.com/microsoft/debugpy/issues/861
+PYTHON_311 = sys.version_info[:2] == (3, 11)
```

Comment:
This should presumably also happen for all versions above 3.11 as well, at least until we can properly fix the underlying issue somehow.

We generally don't use named constants for such checks, and instead just do it in-place:
```py
if sys.version_info >= (3, 11): ...
```

Reply:
Ok, I follow your recommendations

Reply:
May be we can check if `os.path` is a frozen module with that line:

`'<frozen' in os.path.realpath.__code__.co_filename`

But I m not sure this module will be always frozen in next version of Python. 
I have to ask to a python core dev.

Reply:
> I think we can safely assume that this behavior will stay around for at least a few releases.
I agree
> But even if not, the nice thing about -X is that unrecognized switches are still valid, so it would continue working (as a no-op) even if the switch gets retired.
I hope so but I will check